### ISSUE TITLE
Always export static types?

### ISSUE THREAD

**Issue by runeh** [cobraz/generate-runtypes#54]
The static types now look like this:

```
type Something = rt.Static<typeof something>
```

They are not exported by default. The static types are not used inside the generated code, so it might make sense that they are always exported? You'll probably want them exported even if the runtype isn't.

**Comment by cobraz** [comment]
Yeah, we probably want them to be exported as default. Might be some weird cases where you'd like to turn that off though. 🤷‍♂️
